### PR TITLE
[#1017] Introduce the airy create command

### PR DESCRIPTION
**Pull request by paulodiniz**

### CODE REVIEW

File: infrastructure/cli/cmd/create/create.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	provider  string
+	CreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "Creates an instance of the airy core",
```
Comment (lucapette) [inline infrastructure/cli/cmd/create/create.go:13]:
```suggestion
		Short: "Creates an instance of Airy Core",
```

**Review by lucapette — APPROVED** [review]
I'm not sure this "fixes the issue" :D but I think we should merge already as there's nothing wrong with this :)

File: infrastructure/cli/cmd/create/create.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+func init() {
+	CreateCmd.Flags().StringVarP(&provider, "provider", "", "", "One of the supported providers (aws|local)")
+	CreateCmd.MarkFlagRequired("provider")
+	viper.BindPFlag("provider", CreateCmd.PersistentFlags().Lookup("provider"))
+}
+
+func create(cmd *cobra.Command, args []string) {
+	fmt.Println("⚙️  Creating core with provider", provider)
+	fmt.Println("🚀 Starting core with default config")
```
Comment (ljupcovangelski) [inline infrastructure/cli/cmd/create/create.go:27]: Not sure about default config. I would recommend: `Starting core with default components`, as we don't actually provide any configuration for the components by default (the values which are in `airy.yaml`), apart from what is on or off.

File: docs/docs/cli/reference.md [hunk 3]
```diff
@@ -81,6 +81,31 @@ airy config apply [flags]
...
+
+```
+  -h, --help              help for create
+      --provider string   One of the supported providers (aws|local)
+```
+
+#### Options inherited from parent commands
+
+```
+      --apihost string      Airy Core HTTP API host (default "http://api.airy")
```
Comment (chrismatix) [inline docs/docs/cli/reference.md:104]: I'd also split this into `api-host` to keep it consistent

Comment (lucapette) [reply docs/docs/cli/reference.md:104]: I wouldn't do this here. We're reworking the whole config in #1051 and that's the best place for this

**Review by lucapette — APPROVED** [review]

**Review by ljupcovangelski — APPROVED** [review]

**Pull request merged**